tests/msstats_utils.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
```

Comment:
**praise:** I like this refactor :tada:

Reply:
We talked about this but I think it will break inside msstats/R because the contrast matrix get used in the R function "sub-language" ... which is very no-happy with spaces. We can either keep track of the orignal names and put the back in the output files or .... idk ... something else ...